graspologic/match/qap.py

```diff
@@ -400,6 +400,8 @@ def _quadratic_assignment_faq(
 
     # check outlier cases
     if n == 0 or partial_match.shape[0] == n:
+        # Cannot assume partial_match is sorted.
+        partial_match = np.row_stack(sorted(partial_match, key=lambda x: x[0]))
```

Comment:
does sorted require partial_match to be a list?

Reply:
Nope - it can be any iterable.

Reply:
great. can you add a test to the function `test_barycenter_SGM` in test_match.py that gives (a possibly shuffled) full `partial_match` and returns the correct `score_` and `perm_inds`? (Should be very similar to the test lines 138-142)